cost/github/available_seats/available_seats.pt

```diff
@@ -0,0 +1,105 @@
+name "GitHub.com Available Seats Report"
+rs_pt_ver 20180301
+short_description "Gets the number of available seats for a licensed GitHub.com Org and creates an incident if they are out of the policy range."
```

Comment:
Need a `long_description`. It's contents should be: "Version: 1.0"

Reply:
Fixed in 4dd1b8d